src/Tasks/Microsoft.NET.Build.Tasks/AppHost.cs

```diff
@@ -99,18 +100,22 @@ private static int[] ComputeKMPFailureFunction(byte[] pattern)
         }
 
         // See: https://en.wikipedia.org/wiki/Knuth%E2%80%93Morris%E2%80%93Pratt_algorithm
-        private static int KMPSearch(byte[] pattern, byte[] bytes)
+        private static unsafe int KMPSearch(byte[] pattern, MemoryMappedViewAccessor accessor)
         {
             int m = 0;
             int i = 0;
             int[] table = ComputeKMPFailureFunction(pattern);
 
-            while (m + i < bytes.Length)
+            byte* ptreMemoryMappedView = (byte*)0;
```

Comment:
You can use `null` instead of `(byte*)0`

Reply:
fixed, changed to null